providers/linux/testdata/fedora30/etc/os-release

```diff
@@ -0,0 +1,21 @@
+NAME=Fedora
```

Comment:
Some of the files you have added are supposed to be symlinks based on the `ls` output in #57. You can add the file under `testdata/fedora30/usr/lib/os-release`, then add a symlink, and then `git add` the link.

> ls -la /etc/*-release
> lrwxrwxrwx. 1 root root 25 May 16 13:03 /etc/fedora-release -> ../usr/lib/fedora-release
> lrwxrwxrwx. 1 root root 21 May 16 13:03 /etc/os-release -> ../usr/lib/os-release
> lrwxrwxrwx. 1 root root 14 May 16 13:03 /etc/redhat-release -> fedora-release
> lrwxrwxrwx. 1 root root 14 May 16 13:03 /etc/system-release -> fedora-release

Reply:
I made the artifacts in `etc` as soft links.